privacyidea/lib/smsprovider/HttpSMSProvider.py

```diff
@@ -291,10 +276,7 @@ def parameters(cls):
                           "values": ["yes", "no"]
                       },
                       "REGEXP": {
-                          "description": _("Regular expression to modify the phone number "                 
-                                           "to make it compatible with provider. "
-                                           "Enter something like '/[\\+/]//' to remove "
-                                           "pluses and slashes.")
+                          "description": _(ISMSProvider.REGEXP_description)
```

Comment:
I don't know if that works with the translation. You should add the `_(...)` to the string definition in the `ISMSProvider` class.
Also You can just use `cls.REGEXP_description` here since this class inherits from `ISMSProvider`.

Reply:
https://github.com/privacyidea/privacyidea/pull/3597/commits/77a9d792a5a2fa7d0a136c0e2e20cf3a8de76585